services/charts_updater.go

```diff
@@ -1271,7 +1271,6 @@ func BlockTimeAvgChartData() (*types.GenericChartData, error) {
 		ColumnDataGroupingApproximation: "average",
 		TooltipFormatter: `
 		function (tooltip) {
-			this.point.y = Math.round(this.point.y)
```

Comment:
The db is empty but I could test it by faking some values into **rows** after the query above.

Reply:
This change alone does not remove all rounding, does it? See here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/d97a4ac8cd08b81345c836d18874bddd8966ca33/services/charts_updater.go#L1250

Reply:
Yes, this rounds to 2 digits after the comma like etherscan shows it.

Reply:
I see. I was confused by the Jira comments.